contracts/solidity/inhetitance/Main.sol

```diff
@@ -14,8 +14,4 @@ contract Main is Base {
     function returnSuper() public view virtual returns (string memory) {
         return super.classIdentifier();
     }
-
-    function destroyContract(address recipient) public {
```

Comment:
I tried to removed this last time iirc, and Nana told me that we should still keep this until new Cancun EVM release which will remove the support for selfdestruct opcode

Reply:
Not sure how this got into the review.  I may have run a rebase, but I didn't work on that contract.